src/adb/exec.js

```diff
@@ -0,0 +1,15 @@
+let path = require('path');
+let exec = require('mz/child_process').exec;
+let debug = console.log.bind(console, '[adb/exec]');
+let flag = (option, value) => value ? ` ${option} ${value}` : '';
```

Comment:
Sure, I'll split the declarations, and sort them as well.

I could remove the spaces, but if none of the variables are defined, then there would be several empty spaces in the command itself, e.g.  `adb    logcat`.